src/ansys/fluent/core/solver/monitors_manager.py

```diff
@@ -111,10 +110,10 @@ def get_monitor_set_data(
             )
 
     def refresh(self, session_id, event_info) -> None:
-        """Monitors refresh callback.
+        """Monitors the refresh callback.
```

Comment:
He's used a very shorthand description here but the new version is not what he is saying. This refresh method is called whenever either of the events list below occur. This is described as a callback because it is registered to be called on such occurrences.